packages/nimble-blazor/package.json

```diff
@@ -44,7 +44,7 @@
     "@ni/eslint-config-javascript": "^4.2.0",
     "@ni/nimble-components": "*",
     "@ni/nimble-tokens": "*",
-    "playwright": "1.36.0",
+    "playwright": "1.38.0",
```

Comment:
This happened last time too and is the reason it's pinned.
The nuget package (1.38.0) does not have the latest version that matches npm (1.37.1):  https://www.nuget.org/packages/Microsoft.Playwright#readme-body-tab

@jattasNI I manually triggered renovate from the Dependency Dashboard issue just to try it. But yea maybe we shouldn't manage playwright this way because of the blazor behavior. Can see how it's causing a failure because 1.38.0. nuget is not published:
https://github.com/ni/nimble/actions/runs/6201782991/job/16839183495#step:10:1495

Reply:
And from the playwright release notes above (super handy to be inlined like that!) it doesn't look like playright automatically downloads browsers anymore:

> Breaking Changes: Playwright no longer downloads browsers automatically

Yes.. 1.37 -> 1.38 should include breaking changes 🙄

Reply:
Renovate seems to be keeping the PR up-to-date by tracking changes to main. Which I guess is good but triggers a bunch of builds. So closing for now.

Reply:
Created PR #1545 to work around it for now and Tech Debt #1544 to fix the breaking changes.